### ISSUE TITLE
[Bug]: Assertion failed at line 4739 of D:\a\OpenTTD\OpenTTD\src\station_cmd.cpp: runtime > 0

### ISSUE THREAD

**Issue by FilmBoy84** [OpenTTD/OpenTTD#10705]
### Version of OpenTTD

20230423-master-ge9f6bbada2

### Expected result

Game should not crash with the following report

### Actual result

Whilst unpausing the game from initial route planning and committing engines, the attached error appears within a few ticks

Save, dumps and config attached (for GRF list and settings)

![image](https://user-images.githubusercontent.com/33913088/233978965-1f1a5a06-1e73-4b0f-b487-59522a7b7d2a.png)
[20230424_Crash.zip](https://github.com/OpenTTD/OpenTTD/files/11309754/20230424_Crash.zip)

### Steps to reproduce

Load attached save game
Allow time to pass (may require ultraspeed)

**Comment by glx22** [comment]
```
 	[Code externe]	
>	openttd.exe!raise(int signum) Ligne 547	C++
 	openttd.exe!abort() Ligne 71	C++
 	openttd.exe!error(const char * s, ...) Ligne 163	C++
 	openttd.exe!FlowStat::ScaleToMonthly(unsigned int runtime) Ligne 4739	C++
 	openttd.exe!FlowMapper::Run(LinkGraphJob & job) Ligne 54	C++
 	openttd.exe!LinkGraphSchedule::Run(LinkGraphJob * job) Ligne 91	C++
 	openttd.exe!StartNewThread::__l3::<lambda>(const char * name, void(*)(LinkGraphJob *) && F, LinkGraphJob * && <A_0>) Ligne 65	C++
 	[Code externe]	
 	openttd.exe!thread_start<unsigned int (__cdecl*)(void *),1>(void * const parameter) Ligne 97	C++
 	[Code externe]	
```

**Comment by PeterN** [comment]
Thanks for your report.

Resolved by #10707
